### PR TITLE
EVG-3143 Add command to serialize expansions to a file

### PR DESCRIPTION
**Pull request by bsamek**

### CODE REVIEW

File: command/expansion_write.go [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+	"github.com/evergreen-ci/evergreen/model"
+	"github.com/evergreen-ci/evergreen/rest/client"
+	"github.com/mitchellh/mapstructure"
+	"github.com/pkg/errors"
+	yaml "gopkg.in/yaml.v2"
+)
+
+type expansionsWriter struct {
+	File string `mapstructure:"file"`
```
Comment (richardsamuels) [inline command/expansion_write.go]: Enable expansions?

File: command/expansion_write.go [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+	err := mapstructure.Decode(params, c)
+	if err != nil {
+		return errors.Wrap(err, "couldn't decode params")
+	}
+
+	return nil
+}
+
+func (c *expansionsWriter) Execute(ctx context.Context,
+	comm client.Communicator, logger client.LoggerProducer, conf *model.TaskConfig) error {
```
Comment (richardsamuels) [inline command/expansion_write.go]: comm and logger should be _

File: command/expansion_write.go [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+	out, err := yaml.Marshal(conf.Expansions)
+	if err != nil {
+		return errors.Wrap(err, "error marshaling expansions")
+	}
+	if err := ioutil.WriteFile(c.File, out, 0600); err != nil {
+		return errors.Wrapf(err, "error writing expansions to file (%s)", c.File)
+	}
```
Comment (richardsamuels) [inline command/expansion_write.go]: Maybe just log a message: "expansions written to {file}"?

File: command/expansion_write.go (hunk 1 above)
Comment (bsamek) [reply command/expansion_write.go]: done

File: command/expansion_write.go (hunk 2 above)
Comment (bsamek) [reply command/expansion_write.go]: done

File: command/expansion_write.go (hunk 3 above)
Comment (bsamek) [reply command/expansion_write.go]: done

**Comment by bsamek** [comment]
> I think that we should not include private project expansions in this dump unless explicitly requested in a flag.

@tychoish done

**Pull request merged**